### PR TITLE
🔨 Add _NCS.Vehicles:getClosest & _NCS.Vehicles:getVehicles & _NCS.Vehicles:getModel

### PR DESCRIPTION
**Pull request by dezel-2109**
Hi, I try my best.
I put function EnumerateEntities() in getVehicles function files, if you want to move him you can.

### CODE REVIEW

**Comment by dezel-2109** [comment]
And in the first commit I put all in the wrong file but in the last all is good (sorry for my bad english)

File: src/api/vehicles/client/getClosestVehicle.lua [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---getClosest
+---@param coords number
+---@return number, number
+---@public
+
+function API_Vehicles:getClosest(coords)
+
```
Comment (MoskalykA) [inline src/api/vehicles/client/getClosestVehicle.lua:7]: Spaces at the end and beginning of functions are not so useful

File: src/api/vehicles/client/getClosestVehicle.lua [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        local distance = GetDistanceBetweenCoords(vehicleCoords, coords.x, coords.y, coords.z, true)
+
+        if closestDistance == -1 or closestDistance > distance then
+            closestVehicle = vehicles[i]
+            closestDistance = distance
+        end
+    end
+
+    return (closestVehicle), (closestDistance)
+
```
Comment (MoskalykA) [inline src/api/vehicles/client/getClosestVehicle.lua:28]: Spaces at the end and beginning of functions are not so useful

File: src/api/vehicles/client/getClosestVehicle.lua [hunk 3]
```diff
@@ -0,0 +1,29 @@
+---getClosest
+---@param coords number
```
Comment (MoskalykA) [inline src/api/vehicles/client/getClosestVehicle.lua:2]: This is not a number but a table

File: src/api/vehicles/client/getVehicles.lua [hunk 4]
```diff
@@ -0,0 +1,46 @@
+---getVehicles
+---@return number, number
```
Comment (MoskalykA) [inline src/api/vehicles/client/getVehicles.lua:2]: This is not a number but a table

**Review by MoskalykA — CHANGES_REQUESTED** [review]

File: src/api/vehicles/client/getVehicles.lua (hunk 4 above)
Comment (dezel-2109) [reply src/api/vehicles/client/getVehicles.lua:2]: ok I change it

Comment (dezel-2109) [reply src/api/vehicles/client/getVehicles.lua:2]: thx !

**Comment by dezel-2109** [comment]
It's good

**Review by MoskalykA — APPROVED** [review]

**Pull request merged**